sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Fixed a parameter type mismatch in ETW #7 (ReceiveComplete) which caused the duration argument of the operation to be interpreted as a Unicode string and fail to render properly in the formatted message.
```

Comment:
will that hash 7 cause a link?

Reply:
It doesn't when rendered, but the PR tries to be helpful!     The [core package changelog](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md) has the same message and no link.